packages/amesos2/CMakeLists.txt

```diff
@@ -61,11 +62,6 @@ TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_TIMERS
 #
 
 
-IF(${PACKAGE_NAME}_ENABLE_Cholmod)
-  SET(HAVE_AMESOS2_CHOLMOD ON )
```

Comment:
No need for this.  TriBITS automatically defines and sets this var to the correct value.  See:

* https://tribits.org/doc/TribitsDevelopersGuide.html#how-to-add-a-new-tribits-tpl-dependency